items/REFINED_TUNGSTEN.json

```diff
@@ -1,15 +1,17 @@
 {
   "itemid": "minecraft:skull",
   "displayname": "§5Refined Tungsten",
-  "nbttag": "{HideFlags:254,SkullOwner:{Id:\"5d4caed7-a042-3bd7-8e5f-2df71f4d3f5f\",Properties:{textures:[0:{Value:\"ewogICJ0aW1lc3RhbXAiIDogMTcwNzYxNzM1MTUwMSwKICAicHJvZmlsZUlkIiA6ICJiMTM1MDRmMjMxOGI0OWNjYWFkZDcyYWVhYmMyNTQ1MCIsCiAgInByb2ZpbGVOYW1lIiA6ICJUeXBrZW4iLAogICJzaWduYXR1cmVSZXF1aXJlZCIgOiB0cnVlLAogICJ0ZXh0dXJlcyIgOiB7CiAgICAiU0tJTiIgOiB7CiAgICAgICJ1cmwiIDogImh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvNTNmZDhlODdkZDkwMzJjZDcyZjZmYTJkNmI0NWU1M2YwODI3YjUyNzI2M2RhNjQ4ZTBlZTU1NTJjZGFkNmNiYyIsCiAgICAgICJtZXRhZGF0YSIgOiB7CiAgICAgICAgIm1vZGVsIiA6ICJzbGltIgogICAgICB9CiAgICB9CiAgfQp9\"}]}},display:{Lore:[0:\"§5§lEPIC\"],Name:\"§5Refined Tungsten\"},ExtraAttributes:{id:\"REFINED_TUNGSTEN\"}}",
+  "nbttag": "{HideFlags:254,SkullOwner:{Id:\"5d4caed7-a042-3bd7-8e5f-2df71f4d3f5f\",Properties:{textures:[0:{Value:\"ewogICJ0aW1lc3RhbXAiIDogMTcwNzYxNzM1MTUwMSwKICAicHJvZmlsZUlkIiA6ICJiMTM1MDRmMjMxOGI0OWNjYWFkZDcyYWVhYmMyNTQ1MCIsCiAgInByb2ZpbGVOYW1lIiA6ICJUeXBrZW4iLAogICJzaWduYXR1cmVSZXF1aXJlZCIgOiB0cnVlLAogICJ0ZXh0dXJlcyIgOiB7CiAgICAiU0tJTiIgOiB7CiAgICAgICJ1cmwiIDogImh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvNTNmZDhlODdkZDkwMzJjZDcyZjZmYTJkNmI0NWU1M2YwODI3YjUyNzI2M2RhNjQ4ZTBlZTU1NTJjZGFkNmNiYyIsCiAgICAgICJtZXRhZGF0YSIgOiB7CiAgICAgICAgIm1vZGVsIiA6ICJzbGltIgogICAgICB9CiAgICB9CiAgfQp9\"}]}},display:{Lore:[0:\"§8Collection Item\",1:\"\",2:\"§5§lEPIC\"],Name:\"§5Refined Tungsten\"},ExtraAttributes:{id:\"REFINED_TUNGSTEN\"}}",
   "damage": 3,
   "lore": [
+    "§8Collection Item",
+    "",
     "§5§lEPIC"
   ],
   "internalname": "REFINED_TUNGSTEN",
-  "crafttext": "Requires: Tungsten III & HotM 7",
+  "crafttext": "Requires: Tungsten III \u0026 HotM 7",
```

Comment:
Why?

Reply:
I truly do not know, I'll revert it

Reply:
Fixed, in game editor did something weird